analog-commander/usr/share/asteroid-launcher/watchfaces/analog-commander.qml

```diff
@@ -0,0 +1,499 @@
+        }
+        width: parent.width * 0.12
+        height: parent.height * 0.12
+
+        Text {
+            z: 2
+            id: asteroidSlogan
+            font.pixelSize: parent.height * 0.28
+            font.family: "Raleway"
+            color: "white"
+            horizontalAlignment: Text.AlignHCenter
+            anchors {
+                centerIn: parent
+                verticalCenterOffset: -parent.height * -1.000
+            }
+            text: "<b>AsteroidOS</b><br>Hack Your Wrist"
+        }
+
+        MouseArea {
+            anchors.fill: parent
+            onPressAndHold: asteroidLogo.visible ?
+                                asteroidLogo.visible = false :
+                                asteroidLogo.visible = true
+        }
+    }
+
+    Item {
+        id: monthBox
+        property var month: wallClock.time.toLocaleString(Qt.locale(), "mm")
+        onMonthChanged: monthArc.requestPaint()
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.width * 0.00
+            horizontalCenterOffset: -parent.width * 0.22
+        }
+        width: parent.width * 0.22
+        height: parent.height * 0.22
+
+        Canvas {
+            z: 1
+            id: monthArc
+            opacity: !displayAmbient ? 1 : 0.3
+            anchors.fill: parent
+            smooth: true
+            onPaint: {
+                var ctx = getContext("2d")
+                ctx.reset()
+                ctx.beginPath()
+                ctx.fillStyle = "#00ffffff"
+                ctx.arc(parent.width / 2,
+                        parent.height / 2,
+                        parent.width * 0.45,
+                        270 * rad,
+                        360,
+                        false);
+                ctx.strokeStyle = "#77ffffff"
+                ctx.lineWidth = root.height * 0.002
+                ctx.stroke()
+                ctx.fill()
+                ctx.closePath()
+                ctx.lineWidth = root.height * 0.005
+                ctx.lineCap="round"
+                ctx.strokeStyle = "#ff029cdb"
+                ctx.beginPath()
+                ctx.arc(parent.width / 2,
+                        parent.height / 2,
+                        parent.width * 0.456,
+                        270 * rad,
+                        ((wallClock.time.toLocaleString(Qt.locale(),"MM") / 12 * 360) + 270) * rad,
+                        false);
+                ctx.stroke()
+                ctx.closePath()
+            }
+        }
+
+        Repeater {
+            model: 12
+            Text {
+                z: 2
+                id: monthStrokes
+                property bool currentMonthHighlight: Number(wallClock.time.toLocaleString(Qt.locale(), "MM")) === index ||
+                                                     Number(wallClock.time.toLocaleString(Qt.locale(), "MM")) === index + 12
+                antialiasing: true
+                font.pixelSize: currentMonthHighlight ?
```

Comment:
When you do multiple things with a `font` like this, I'd suggest using `font {` instead and always listing the family first for consistency.  So this might be better as:
```
font {
   family: "Teko"
   pixelSize: root.height * (currentMonthHighlight ? 0.36 : 0.03)
   // etc.
}
```